### DISCUSSION TITLE
EDGE_FUNCTION_INVOCATION_TIMEOUT

### DISCUSSION THREAD

**Discussion by eikyu721** [timqian/openprompt.co#42] [Q&A] [answered]
会话中经常遇到
```
An error occurred with your deployment

EDGE_FUNCTION_INVOCATION_TIMEOUT
```
是因为API当前服务的人太多导致触发API限制，还是因为一次不能进行太多次对话，或者是其他原因？

**Comment by timqian** [comment, marked as answer]
感谢反馈，我们也注意到了这个问题，一般是因为聊天记录总字数太多导致的，为了记住上下文，需要把整个聊天记录发送给服务端，所以如果聊天记录比较长，容易触发字数限制。

我们后面会优化长聊天的体验和报错信息。

**Reply by EarlBart** [reply]
现在是6月底，还没好。网站是不是被攻击了？

**Reply by timqian** [reply]
感谢反馈，确实是被攻击了，现已修复
